File: Scripts/server/file-input-output/javascript/file-input-output.js
var r = 1;

function executeTask(i) {

	fs = require('fs');
	var contents = fs.readFileSync('../test_directory/' + i, 'utf8');
	fs.writeFileSync('output.txt',contents);
	return i;
}

for ( var i = 1; i < 10000; i++) {
	r = executeTask(i);
}

File: Scripts/volatile_and_force_compiler_task/file-input-output/c-sharp/file-input-output.cs
class ReadFromFile
{

    private static volatile int r = 1;

    public static int executeTask(int i) {

	string str = System.IO.File.ReadAllText(@"../test_directory/"+i);
	System.IO.File.WriteAllText(@"output.txt", str);
	return i + 1;
    }
	
    static void Main()
    {
	for ( int i = 1; i <= 10000; ++i ) {
		r = executeTask(i);
	}
    }
}
